test/client.spec.js

```diff
@@ -87,42 +87,4 @@ describe('Client', function() {
             expect(baseParams.api_key).to.equal(MOCK_KEY);
         });
     });
-
```

Comment:
Mocking the `request-promise` effort is unnecessarily tasking, and would require an additional library (`mockery`). Given this is simply a test to make sure this is called, running any of the demos should provide sufficient "coverage"